### PR TITLE
US7409 add group leader microclient

### PR DESCRIPTION
**Pull request by silbermm**
* added all needed files for displaying microclient at /group-leader
* commented out broken SharedHeader import in app.js

### CODE REVIEW

File: apps/crossroads_interface/web/static/js/app.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { SharedHeader } from 'crds-shared-header/dist/bundle';
+// import { SharedHeader } from 'crds-shared-header/dist/bundle';
```
Comment (richardbarker) [inline apps/crossroads_interface/web/static/js/app.js]: We need this still.

Comment (silbermm) [inline apps/crossroads_interface/web/static/js/app.js]: I'll uncomment, it was just broken at the time I was using it

File: apps/crossroads_interface/web/controllers/crds_group_leader_controller.ex [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  def index(conn, _params) do
+    render conn, "app_root.html", %{ "js_files": [
+        "/js/group-leader/inline.bundle.js",
+        "/js/group-leader/polyfills.bundle.js",
+        "/js/group-leader/styles.bundle.js",
+        "/js/group-leader/vendor.bundle.js",
+        "/js/group-leader/main.bundle.js"
+      ], "base_href": "/group-leader" }
+  end
```
Comment (richardbarker) [inline apps/crossroads_interface/web/controllers/crds_group_leader_controller.ex]: This is more of a stylistic question, but should we be using a plug for base_href or just passing it in?  Works either way.  I wonder if you have a preference one way or another

Comment (silbermm) [inline apps/crossroads_interface/web/controllers/crds_group_leader_controller.ex]: I'm not sure which is better. I changed it to the plug. If we had multiple actions in here, a plug makes more sense, but we will almost always only have one action per microclient.

**Pull request merged**